protocols/twitter/twitter.c

```diff
@@ -699,6 +700,9 @@ static void twitter_logout(struct im_connection *ic)
 		g_free(td->prefix);
 		g_free(td->url_host);
 		g_free(td->url_path);
+		for (i=0; i<TWITTER_LOG_LENGTH; i++)
+			if (td->log[i].text)
```

Comment:
Please use braces in all blocks, even when they can be omitted. This applies to other places too.